src/ripple/test/jtx/Env.h

```diff
@@ -182,7 +182,7 @@ class Env
         JTx jt(std::forward<JsonValue>(jv));
         invoke(jt, fN...);
         autofill(jt);
-        return jt;
+        return std::move(jt);

```

Comment:
I don't believe the move is necessary here (I believe is actually inhibits RVO here). However, you clearly made this edit, so I might be wrong...